apps/nextjs/src/pages/payment/success.tsx

```diff
@@ -1,32 +1,19 @@
 import { type NextPage } from "next";
-import Image from "next/image";
-import { useEffect } from "react";
-import { env } from "../../env/client.mjs";
+import { Card } from "../../components/card.jsx";
```

Comment:
```suggestion
import { Card } from "../../components/card";
```

The .jsx extension didn't work for me